images/win/scripts/SoftwareReport/SoftwareReport.Helpers.psm1

```diff
@@ -1,31 +1,3 @@
-function Build-MarkdownElement
-{
-    <#
-    .SYNOPSIS
-    Build markdown element for cached tool.
-
-    .DESCRIPTION
-    Build markdown element that contains name of tool, set of versions and additional notes.
-
-    .PARAMETER Head
-    Header of cached tool markdown element
-
-    .PARAMETER Content
-    Array of lines that contains required information about installed tool instances.
-    #>
-
-    param
-    (
-        [String] $Head,
-        [Object[]] $Content
-    )
-
-    $markdown = New-MDHeader $Head -Level 4
-    $markdown += New-MDParagraph -Lines $Content -NoNewLine
-
-    return $markdown
-}
-
 function Get-CachedToolInstances
```

Comment:
could you please check if this function is used anywhere and remove if not. I think I forgot to remove it 🤦

Reply:
yes, it is not used - removed